parsel/utils.py

```diff
@@ -80,4 +81,12 @@ def extract_regex(regex, text, replace_entities=True):
     strings = flatten(strings)
     if not replace_entities:
         return strings
-    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
\ No newline at end of file
+    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
+
+def selector_data_shorten(data, width):
+    """Shortens the preview of extracted data by adding placeholder '...'
+    """
+    if six.PY34:
+        return textwrap.shorten(data, width=width, placeholder="...")
+    else:
+        return data[:40]
```

Comment:
This should use the `width` argument, instead of hardcode it to `40` here.
We also want it to mimic the behavior of `shorten`, like:

    return data[:width - 3] + "..." if (len(data) > width - 3) else return data

Otherwise, the issue will remain in Python 2.